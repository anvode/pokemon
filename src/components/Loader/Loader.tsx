import * as React from 'react';

import './Loader.scss';

const Loader: React.FC = () => {

    return <>
        <h1 className="app__title">Loading</h1>
        <div className="loader__container">
            <div className="loader"><div></div><div></div><div></div><div></div></div>
        </div>
    </>;
};

export default Loader;
