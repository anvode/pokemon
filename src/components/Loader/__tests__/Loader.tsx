import * as React from 'react';
import { render, cleanup } from '@testing-library/react';
import Loader from '../Loader';

afterEach(cleanup);

describe('Loader', () => {
    it('renders without crashing', () => {
        const { asFragment } = render(<Loader />);
        expect(asFragment()).toMatchSnapshot();
    });
});

