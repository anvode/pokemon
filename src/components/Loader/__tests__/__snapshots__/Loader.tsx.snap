// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders without crashing 1`] = `
<DocumentFragment>
  <h1
    class="app__title"
  >
    Loading
  </h1>
  <div
    class="loader__container"
  >
    <div
      class="loader"
    >
      <div />
      <div />
      <div />
      <div />
    </div>
  </div>
</DocumentFragment>
`;
